### PR TITLE
Prefer hosts at least twice the size of the node

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

**Comment by bratseth** [comment]
@freva I merged this by accident. Please review though.

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/PrioritizableNode.java [hunk 1]
```diff
@@ -128,6 +133,15 @@ public int compareTo(PrioritizableNode other) {
     /** Returns the allocation skew of the parent of this after adding this node to it */
     double skewWithThis() { return skewWith(node.resources()); }
 
+    private boolean lessThanHalfTheHost(PrioritizableNode node) {
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/PrioritizableNode.java:136]: The method name is the opposite of what it returns

**Review by freva — COMMENTED** [review]
:+1:

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/PrioritizableNode.java (hunk 1 above)
Comment (bratseth) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/PrioritizableNode.java:136]: No.

**Pull request merged**